impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -156,8 +156,8 @@ GeometryResult PointFieldGeometry::GetPositionBufferGPU(
       host_buffer.Emplace(points_.data(), points_.size() * sizeof(Point),
                           DefaultUniformAlignment());
 
-  BufferView geometry_buffer =
-      host_buffer.Emplace(nullptr, total * sizeof(Point), alignof(Point));
+  BufferView geometry_buffer = host_buffer.Emplace(
+      nullptr, total * sizeof(Point), DefaultUniformAlignment());
```

Comment:
This should be the max of the alignof the Point and the default uniform alignment right?

Reply:
Done in https://github.com/flutter/engine/pull/51341